File: Portal-App/app/src/main/java/dev/alyxia/portal_app/rest/dto/Product.kt
package dev.alyxia.portal_app.rest.dto

import dev.alyxia.portal_app.lib.DateSerializer
import kotlinx.serialization.Serializable
import java.util.*

typealias ProductList = List<ApiProduct>

@Serializable
data class ApiProduct(
    @Serializable(with = DateSerializer::class)
    val created_at: Date,
    @Serializable(with = DateSerializer::class)
    val updated_at: Date,

    val id: Int,
    val name: String,
    val description: String,
    val price: Int
)

File: Dashboard/DTO/Product.cs
namespace Dashboard.DTO;

public class Product
{
    public int id { get; set; }
    public string name { get; set; }
    public string description { get; set; }
    public string price { get; set; }
    public bool spanish { get; set; }
    public bool warm { get; set; }
    public bool cold { get; set; }
    public bool vega { get; set; }
    public string created_at { get; set; }
    public string updated_at { get; set; }
}
